pyproject.toml

```diff
@@ -50,7 +50,7 @@ flake8-use-fstring = "^1.1"
 isort = "^5.10.1"
 mccabe = "^0.7.0"
 pre-commit = "^4.0.0"
-pyright = "^1.1.267"
+pyright = "==1.1.402"
```

Comment:
Is the idea to bump pyright versions manually via "bump PRs" and fix pyright issues in the same PR?

Reply:
Yes, exactly. Even minor versions for pyright can introduce changes that break our ci.